packages/blocks/src/list-block/list-block.ts

```diff
@@ -120,6 +113,21 @@ export class ListBlockComponent extends BlockElement<
     bindContainerHotkey(this);
 
     this._inlineRangeProvider = getInlineRangeProvider(this);
+
+    this._updateFollowingListSiblings();
+    this.disposables.add(
+      this.model.childrenUpdated.on(() => {
+        this._updateFollowingListSiblings();
+      })
+    );
+    this.host.std.doc.slots.blockUpdated.on(e => {
```

Comment:
Is this change also needed to be put in disposables?

Reply:
nice catch